providers/flagd/README.md

```diff
@@ -16,6 +16,9 @@ flagd.WithHost(string)                  // defaults to localhost
 flagd.WithPort(uint16)                  // defaults to 8013
 flagd.FromEnv()                         // sets the provider configuration from environment variables
 flagd.WithSocketPath(string)            // no default, when set a unix socket connection is used (only available for GRPC)
+flagd.WithoutCache()                    // disables caching of flag evaluations
+flagd.WithLRUCache(1000)                // enables LRU caching (see configuring caching section)
+flagd.WithBasicInMemoryCache()          // enables basic in memory cache (see configuring caching section)
```

Comment:
might it be worthwhile to add a footnote here to mention that the latest option will take precedence, although as previously mentioned this may be overkill, e.g. 
`flagd.NewProvider(flagd.WithoutCache(), flagd.WithLRUCache(1000))`

Reply:
Imo this is overkill, I can't imagine a scenario where a user would supply both options